RFC/src/RFC-0110_BaseNodes.md

```diff
@@ -196,6 +196,23 @@ verification is achieved by
 When Base Nodes receive blocks from peers while synchronizing, the usual
 [block validation](#block-validation-and-propagation) process is followed.
 
+### Pruning and cut-through
+[Pruning and cut-through]: #Pruning-and-cut-through "Remove allready spent outputs from the [utxo]"
+
+In MimbleWimble, only the current [UTXO](utxo) set is of importance. This allows base layer nodes to remove old used inputs from the [blockchain] and or the [mempool]. [Cut-through](cut-through) happens in the [mempool] while pruning happens in the [blockchain] with already confirmed transactions. This will remove some of the inputs and outputs, but will retain the excesses  of each [transaction]. 
+
+Pruning is only for the benefit of the local base node so this is an optional, but recommended requirement. A Base node will either run in archive mode or prune mode, if the base node is running in archive mode it should not prune. 
+
+[base node]s has the following **optional** reponsibilities:
+
+1. SHOULD search for used outputs in old blocks when a new block is received from another [base node].
+2. SHOULD apply pruning to each transaction found.
+
+[base node]s have the following **required** reponsibilities:
+
+1. MUST check if a newly received block for [cut-through]. If a block contains already spent outputs, reject that block. 
```

Comment:
It is probably a more general requirement, but the same time it is probably better to mention it after we mention what pruning and cut-through is?

I am open to both, or perhaps we move the explaining section higher, and merge all requirements?

Reply:
It just feels to me that the rejection of the block is not actually related to pruning or cut-through? It would happen without pruning or cut-through.

Reply:
I think this is perhaps a more compelling argument